netlogo-gui/src/main/properties/EditPanel.scala

```diff
@@ -130,18 +131,23 @@ class EditPanel(val target: Editable, val compiler: CompilerServices, colorizer:
 
   def valid() = {
     def valid(editor: PropertyEditor[_]) = {
+      // Exclude runMetricsCondition from checking if the box is empty because an empty input is valid
+      if (editor.accessor.accessString != "runMetricsCondition") {
```

Comment:
I think it would be clearer to have 

if (string) == "runMetricsCondition" {
   true
} else {
previous stuff
}